cmd/clusterctl/cmd/move.go

```diff
@@ -98,7 +105,38 @@ func runMove() error {
 		return errors.New("please specify a target cluster using the --to-kubeconfig flag when not using --dry-run, --to-directory or --from-directory")
 	}
 
-	c, err := client.New(ctx, cfgFile)
+	configClient, err := config.New(ctx, cfgFile)
+	if err != nil {
+		return err
+	}
+
+	clientOptions := []client.Option{}
+	if mo.hideAPIWarnings {
+		clientOptions = append(clientOptions,
+			client.InjectClusterClientFactory(
+				func(input client.ClusterClientFactoryInput) (cluster.Client, error) {
+					return cluster.New(
+						cluster.Kubeconfig(input.Kubeconfig),
+						configClient,
+						cluster.InjectYamlProcessor(input.Processor),
+						cluster.InjectProxy(
+							cluster.NewProxy(
+								cluster.Kubeconfig(input.Kubeconfig),
+								cluster.InjectWarningHandler(
+									apiwarnings.DefaultHandler(
+										logf.Log.WithName("API Server Warning"),
```

Comment:
Previously, we used the default warning handler provided by controller-runtime. It uses the name `KubeAPIWarningLogger` to log messages.

If we want to keep our output unchanged, I can use `KubeAPIWarningLogger`, instead of `API Server Warning`, here.

Reply:
I'm fine with the new name